src/Uno.UI/UI/Xaml/Media/ImageSource.netstd.cs

```diff
@@ -66,16 +67,12 @@ internal IDisposable Subscribe(Action<ImageData> onSourceOpened, int? targetWidt
 		#region Implementers API
 		private protected virtual bool TryOpenSourceSync(int? targetWidth, int? targetHeight, out ImageData image)
 		{
-#if __NETSTD_REFERENCE__
 			image = default;
 			return false;
-#else
-			// Unlike other platforms, on WASM all the legacy sources are handled synchronously.
-			return TryOpenSourceLegacy(out image);
-#endif
 		}
 
-		private protected virtual bool TryOpenSourceAsync(int? targetWidth, int? targetHeight, out Task<ImageData> asyncImage)
+		private protected virtual bool TryOpenSourceAsync(CancellationToken ct, int? targetWidth, int? targetHeight,
```

Comment:
Please, either all parameters each on a new line, either all on the same with the method name